app/src/main/java/com/keylesspalace/tusky/components/occurrence/OccurrenceActivity.kt

```diff
@@ -0,0 +1,231 @@
+/* Copyright Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>.
+ */
+
+package com.keylesspalace.tusky.components.occurrence
+
+import android.content.Context
+import android.content.Intent
+import android.graphics.Color
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.ListAdapter
+import com.google.android.material.color.MaterialColors
+import com.keylesspalace.tusky.BaseActivity
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.databinding.ActivityOccurrencesBinding
+import com.keylesspalace.tusky.databinding.ItemOccurrenceBinding
+import com.keylesspalace.tusky.db.AccountEntity
+import com.keylesspalace.tusky.db.AppDatabase
+import com.keylesspalace.tusky.di.Injectable
+import com.keylesspalace.tusky.di.ViewModelFactory
+import com.keylesspalace.tusky.util.BindingHolder
+import com.keylesspalace.tusky.util.getDurationStringAllowMillis
+import com.keylesspalace.tusky.util.getRelativeTimeSpanString
+import com.keylesspalace.tusky.util.viewBinding
+import com.keylesspalace.tusky.util.visible
+import dagger.android.DispatchingAndroidInjector
+import dagger.android.HasAndroidInjector
+import kotlinx.coroutines.runBlocking
+import java.text.DateFormat
+import javax.inject.Inject
+
+class OccurrenceActivity : BaseActivity(), Injectable, HasAndroidInjector {
```

Comment:
I'd call it DebugActivity/DebugLogActivity or something like that

Reply:
Hm, it's not used for debugging.

It should/could have the same name as in the ui (and there "debug" would be somewhat misleading).

It shows (hopefully) significant events in the app life.